vignettes/scoring-rules.Rmd

```diff
@@ -0,0 +1,470 @@
+---
+title: "Scoring rules in `scoringutils`"
+author: "Nikos Bosse"
```

Comment:
in epinowcast we have moved away from tracking vignette authorship as it just gets out of date and is hard to agree on. I don't mind either way here